app/src/main/java/org/openobservatory/ooniprobe/common/CountlyManager.java

```diff
@@ -0,0 +1,56 @@
+package org.openobservatory.ooniprobe.common;
+
+import android.content.Context;
+
+import org.openobservatory.ooniprobe.BuildConfig;
+
+import ly.count.android.sdk.Countly;
+import ly.count.android.sdk.CountlyConfig;
+import ly.count.android.sdk.DeviceId;
+import ly.count.android.sdk.messaging.CountlyPush;
+
+public class CountlyManager {
+    public static String[] basicFeatures = new String[]{
+            ly.count.android.sdk.Countly.CountlyFeatureNames.location,
+    };
+
+    public static String[] analyticsFeatures = new String[]{
+            ly.count.android.sdk.Countly.CountlyFeatureNames.sessions,
+            ly.count.android.sdk.Countly.CountlyFeatureNames.views,
+            ly.count.android.sdk.Countly.CountlyFeatureNames.events
+            //TODO evaluate scrolls, clicks, forms, attribution
+    };
+
+    public static String[] crashFeatures = new String[]{
+            ly.count.android.sdk.Countly.CountlyFeatureNames.crashes,
+    };
+
+    public static String[] pushFeatures = new String[]{
+            ly.count.android.sdk.Countly.CountlyFeatureNames.push,
+    };
```

Comment:
Could all the above `public static String[]` be private instead?

Reply:
Sure